lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -201,6 +201,21 @@ namespace Js
             DynamicType::New(scriptContext, TypeIds_Object, typedArrayPrototype, nullptr,
                 DeferredTypeHandler<InitializeCharArrayPrototype, DefaultDeferredTypeFilter, true>::GetDefaultInstance()));
 
+        if (scriptContext->GetConfig()->IsESSharedArrayBufferEnabled())
+        {
+            sharedArrayBufferPrototype = DynamicObject::New(recycler,
+                DynamicType::New(scriptContext, TypeIds_Object, objectPrototype, nullptr,
+                    DeferredTypeHandler<InitializeSharedArrayBufferPrototype, DefaultDeferredTypeFilter, true>::GetDefaultInstance()));
+
+            sharedArrayBufferType = DynamicType::New(scriptContext, TypeIds_SharedArrayBuffer, sharedArrayBufferPrototype, nullptr,
```

Comment:
>sharedArrayBufferType [](start = 12, length = 21)

Types are initialized in InitializeTypes(), please remove from here. Also move arrayBufferType Line#140 to InitializeTypes().